test/minitest/test_minitest_assertions.rb

```diff
@@ -1366,7 +1371,7 @@ def test_skip_until
     d0 = Time.now
     d1 = d0 + 86_400 # I am an idiot
 
-    assert_output "", /Stale skip_until \"not yet\" at .*?:\d+$/ do
+    assert_output "", /Stale skip_until \"not yet\" at #{__FILE__}:\d+$/ do
```

Comment:
I rather dislike absolute paths and am thinking of changing it... Can you revert this bit or does it serve a purpose I don't see yet?

Reply:
The RegEx won't work with windows paths.  Example `C:\Windows`.  Sorry, I thought I added a comment somewhere...

Reply:
it _should_ work with windows paths... that `$` should force the regexp to keep trying after matching `c:\`, no? We can remove the `?` if that's what's interfering but I didn't think it was.

```
>> re = /Stale skip_until \"not yet\" at .*?:\d+$/
=> /Stale skip_until \"not yet\" at .*?:\d+$/
>> s = "Stale skip_until \"not yet\" at c:\\blah.rb:42"
=> "Stale skip_until \"not yet\" at c:\\blah.rb:42"
>> s =~ re
=> 0
```

Reply:
Sorry. I think I figured out what I did.  The rexgexp works, I tried the change assuming the test was bad.  Actually, the code was bad, so it was fixed in `assertions.rb` in the first commit.  I accidentally left the test change in.

It's only an issue with Windows, and although `skip_until` makes sense, I don't recall seeing it in use.

Anyway, removed the change.